services/client/src/test/java/org/cryptimeleon/incentive/client/integrationtest/AliveTest.java

```diff
@@ -2,45 +2,56 @@
 
 import org.cryptimeleon.incentive.client.BasketClient;
 import org.cryptimeleon.incentive.client.CreditClient;
+import org.cryptimeleon.incentive.client.InfoClient;
 import org.cryptimeleon.incentive.client.IssueClient;
 import org.junit.jupiter.api.Test;
-import org.springframework.web.reactive.function.client.WebClient;
+
+import java.time.Duration;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class AliveTest extends IncentiveSystemIntegrationTest {
 
-    /*
+    /**
      * Integration test that queries the hello world endpoint
```

Comment:
Comment outdated or am I missing something?

Reply:
No, the `/` endpoint returns with "Hello from ... service!", hence I called it hello world endpoint :)
The purpose is to have a simple test that checks whether the service is reachable under the correct url.